cmd/main.go

```diff
@@ -142,7 +165,18 @@ func main() {
 		zap.Duration("stream_success_period_auth", envConfig.StreamSuccessPeriod.Auth),
 	)
 
-	// Start the gops agent
+	// List pods from default namespace
+	pods, err := clientset.CoreV1().Pods("illumio-cloud").List(context.TODO(), metav1.ListOptions{})
+	if err != nil {
+		klog.Error(err, "Failed to list pods")
+		return
+	}
+
+	for _, pod := range pods.Items {
+		klog.Info("Pod found ", "name ", pod.Name)
+	}
+
+	// Start the gops agent and listen on a specific address and port
```

Comment:
Same here, please remove this test code now.
```suggestion
	// Start the gops agent
```